modules/Bio/EnsEMBL/Variation/Pipeline/SpliceAI/SpliceAI_conf.pm

```diff
@@ -0,0 +1,195 @@
+=head1 LICENSE
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2020] EMBL-European Bioinformatics Institute
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+     http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+=cut
+
+
+=head1 CONTACT
+ Please email comments or questions to the public Ensembl
+ developers list at <http://lists.ensembl.org/mailman/listinfo/dev>.
+ Questions may also be sent to the Ensembl help desk at
+ <http://www.ensembl.org/Help/Contact>.
+=cut
+package Bio::EnsEMBL::Variation::Pipeline::SpliceAI::SpliceAI_conf;
+
+use strict;
+use warnings;
+
+use Bio::EnsEMBL::Hive::PipeConfig::HiveGeneric_conf;
+ # All Hive databases configuration files should inherit from HiveGeneric, directly or indirectly
+use base ('Bio::EnsEMBL::Hive::PipeConfig::EnsemblGeneric_conf');
+
+sub default_options {
+    my ($self) = @_;
+
+    # The hash returned from this function is used to configure the
+    # pipeline, you can supply any of these options on the command
+    # line to override these default values.
+    
+    # You shouldn't need to edit anything in this file other than
+    # these values, if you find you do need to then we should probably
+    # make it an option here, contact the variation team to discuss
+    # this - patches are welcome!
+
+    return {
+        %{ $self->SUPER::default_options()
+        },    # inherit other stuff from the base class
+
+        hive_auto_rebalance_semaphores => 1,
+        hive_default_max_retry_count => 0,
+        hive_force_init      => 1,
+        hive_use_param_stack => 1,
+
+        pipeline_name           => $self->o('pipeline_name'),
+        main_dir                => $self->o('main_dir'), # main_dir = '/hps/nobackup2/production/ensembl/dlemos/tmp_spliceai_pipeline'
+        tmp_split_vcf_dir       => $self->o('main_dir') . '/tmp_split_vcf', # contains vcf input splited without headers
+        tmp_split_vcf_input_dir => $self->o('main_dir') . '/tmp_split_vcf_input', # contains vcf input splited with headers, these are the files used to run SpliceAI
+        tmp_output_dir          => $self->o('main_dir') . '/tmp_output', # temporary output files, still splited
+        output_dir              => $self->o('main_dir') . '/output', # final output files already merged by chromosome
```

Comment:
I would suggest that that you are more clear about what the directories store and try to avoid the use of tmp. What about:
splitted_vcf_no_header_dir
splitted_vcf_input_dir
splitted_vcf_output_dir
output_dir